Gemfile.lock

```diff
@@ -229,9 +227,12 @@ DEPENDENCIES
   sinatra (= 1.4.7)
   slop (= 3.4.5)
   statsd-ruby (~> 1.3.0)
-  test-unit
+  test-unit-minitest (~> 0.9.1)
   timecop (= 0.8.0)
   unf (= 0.1.4)
   unicorn (= 5.1.0)
   webmock (~> 1.24)
   whenever (~> 0.9.4)
+
+BUNDLED WITH
```

Comment:
Just in case, I've removed it when I saw @jackscotti comment. so please don't merge it until I put that back